dependency-check-maven-config.xml

```diff
@@ -1,18 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://www.owasp.org/index.php/OWASP_Dependency_Check_Suppression">
 
```

Comment:
❓ Did we run OWASP to make sure those vulnerabilities are actually gone?

Reply:
I successfully completed an OWASP build locally.